src/pseudopeople/configuration/interface.py

```diff
@@ -14,26 +14,25 @@ def get_config(dataset_name: str = None, overrides: Union[Path, str, Dict] = Non
     """
     Function that returns the pseudopeople configuration,
     including all default values.
-    If :code:`dataset_name` is None (the default), the returned dictionary has exactly
+    If :code:`dataset_name` is None (the default), the returned dictionary includes
+    the configuration for all datasets. If a dataset name is supplied, only that
+    dataset's configuration is returned. In both cases, the returned dictionary has exactly
     the structure described on the :ref:`Configuration page <configuration_main>`.
-    If a dataset name is supplied, only returns the part of the configuration for that
-    dataset, i.e. :code:`get_config(dataset_name)` is equivalent to
-    :code:`get_config()[dataset_name]`.
```

Comment:
It might be worth still specifying that `get_config(dataset_name)` is equivalent to
   `{dataset_name: get_config()[dataset_name]}`

Reply:
confusing!

Reply:
Ok, I thought that might help clarify things, but if it makes it _more_ confusing, let's leave it deleted as Steve has it.